src/Moq/Properties/Resources.Designer.cs

```diff
@@ -9,763 +9,781 @@
 //------------------------------------------------------------------------------
 
 namespace Moq.Properties {
-	using System;
```

Comment:
I know it's annoying, but could you please manually restore indentation to use tabs instead of spaces, so that your PR's diff only includes what actually changed. Thanks!

(I'll eventually get around to fixing this annoying issue.)